.travis.yml

```diff
@@ -1,20 +1,15 @@
+dist: trusty
 language: java
 sudo: false
-install: mvn -Pstaging install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
-script: mvn -Pstaging test -B
+
 jdk:
-  - oraclejdk8
```

Comment:
Are we dropping `jdk8` altogether? I think we should keep that, [Xenial images can still run `openjdk8`](https://github.com/weld/core/blob/master/.travis.yml).